web/client/selectors/importer.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export const isImporterConfigured = state => state.localConfig && state.localConfig.plugins && !!state.localConfig.plugins.importer;
```

Comment:
Maybe we can create some generic selector to look for configured plugins, instead of creating one for each plugin. What do you think?

Reply:
I'd like to do the same as above. Think about pages and do it once, in a separate task. Not so difficult but requires some plan and time.

Reply:
Maybe this could work:

export const isPluginConfigured = (plugin) => ((state) => state.localConfig && state.localConfig.plugins && !!state.localConfig.plugins[plugin]);

Reply:
Where should I place it. `web/client/selectors/{,,,}.js` ?
plugins.js , localConfig.js ? What do you think is a better name for this selector?

Reply:
plugins.js